src/controllers/UserController.ts

```diff
@@ -0,0 +1,28 @@
+import User, { IUser } from '../models/User';
+import { Request, Response } from "express";
+import asyncHandler from 'express-async-handler';
+
+interface UserController {
+    createUser?: any;
+    getUsers?: any;
+}
+
+const userController: UserController = {};
+
+const createUser = asyncHandler(async (req: Request, res: Response) => {
+    console.log("create users in the controller")
+    console.log(req.body)
+    const newDoc = await User.create(req.body);
+    res.status(201).json({ data: newDoc });
+});
+
+const getUsers = asyncHandler(async (req, res, next) => {
+    const document = await User.find();
```

Comment:
Same here with "find"

Reply:
Wouldn't  you need to add a parameter ad the find method?
Or was this part of your TODO's

Reply:
this also comes from the Model